tests/manage/storageclass/test_new_sc_rbd_e2e_workloads.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, tier2
+
+
```

Comment:
Looks like this test covers only the test https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-2345.

where do you cover the following tests?
https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-2347
https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-2346
https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-2344

Reply:
I'll update the parameters and test case IDs